app/angular/src/server/config.js

```diff
@@ -3,6 +3,10 @@ import fs from 'fs';
 import path from 'path';
 import loadBabelConfig from './babel_config';
 import loadTsConfig from './ts_config';
+import {
+  getAngularCliWebpackConfigOptions,
+  applyAngularCliWebpackConfig,
+} from './angular-cli_config';
 
 // avoid ESLint errors
 const logger = console;
```

Comment:
This should use `@storybook/node-logger`

Reply:
@Quramy, I think this is the last comment to take care of